packages/sampler/stories/qa/VirtualListNative.js

```diff
@@ -1,29 +1,56 @@
 import React from 'react';
 import Item from '@enact/moonstone/Item';
-import {VirtualListNative} from '@enact/moonstone/VirtualList';
+import {VirtualListNative, VirtualListBase} from '@enact/moonstone/VirtualList';
 import {Column, Cell} from '@enact/ui/Layout';
+import ri from '@enact/ui/resolution';
+import {ScrollableBaseNative as UiScrollableBaseNative} from '@enact/ui/Scrollable/ScrollableNative';
+import {VirtualListBaseNative as UiVirtualListBaseNative} from '@enact/ui/VirtualList/VirtualListBase';
 
+import {action} from '@storybook/addon-actions';
 import {storiesOf} from '@storybook/react';
 
+import {boolean, number, select} from '../../src/enact-knobs';
+import {mergeComponentMetadata} from '../../src/utils/propTables';
+
+const Config = mergeComponentMetadata('VirtualListNative', UiVirtualListBaseNative, UiScrollableBaseNative, VirtualListBase);
+
+const
+	defaultDataSize = 10,
+	prop = {
+		scrollbarOption: ['auto', 'hidden', 'visible']
+	},
+	wrapOption = {
+		false: false,
+		true: true,
+		'&quot;noAnimation&quot;': 'noAnimation'
+	};
+
 storiesOf('VirtualListNative', module)
 	.add(
 		'with extra items',
 		() => (
 			<Column>
 				<Cell
 					component={VirtualListNative}
-					focusableScrollbar
+					dataSize={number('dataSize', Config, defaultDataSize)}
+					direction="vertical"
+					focusableScrollbar={boolean('focusableScrollbar', Config)}
+					horizontalScrollbar={select('horizontalScrollbar', prop.scrollbarOption, Config)}
 					// eslint-disable-next-line react/jsx-no-bind
 					itemRenderer={({index, ...rest}) => {
-						const text = `Item ${('00' + index).slice(-3)}`;
+						const text = 'Item ' + index;
```

Comment:
Is there any reason to change this line? It will change the item text.

Reply:
That's because I added a knob for `dataSize` prop. In order to show the well-ordering number with presenting a digit number in each item, we should add some calculation functions and data to store the results. But I think it does not matter how to show the item contents depending on a digit number. I've tried to make them simple.

Reply:
There is no absolute answer for your question. But I hope we could have same patterns to write our sampler code. With this reason, let's change it to `('Item ' + index).slice(-itemNumberDigits)` like other samplers.